### ISSUE TITLE
Display of GFI problem

### ISSUE THREAD

**Issue by kgaeb** [geosolutions-it/austrocontrol-C125#299]
In the current installation the GFI uses the freemarker template serviced by our GeoServer and looks in the MapStore like this:
![image](https://user-images.githubusercontent.com/34339627/190403680-8478280e-57f0-4dc0-8f9d-e940d48c2b6f.png)

In the new version the same GFI is displayed like the following:
![image](https://user-images.githubusercontent.com/34339627/190404111-db73983b-64c8-4ecc-8500-8f4bacd026b8.png)

As is shown on the second image the content of the GFI is shown in disordered form.
The content delivered from GeoServer is the same for both.

Also other GFI (here from AD):
![image](https://user-images.githubusercontent.com/34339627/190407803-5121238b-7824-4808-bfff-6f9b5cb1f1ea.png)
The symbols should be on the left side and it seems there is somewhere a line break missing, because the symbols are on the wrong line

**Comment by tdipisa** [comment]
@kgaeb 

>As is shown on the second image the content of the GFI is shown in disordered form.
The content delivered from GeoServer is the same for both.

> The symbols should be on the left side and it seems there is somewhere a line break missing, because the symbols are on the wrong line

It seems quite strange indeed. MS simply renders the GFI result as it is returned by GS. We will do a check of course but I suspect that there could be something to fix in the free marker template maybe depending on the selected feature.

**Comment by kgaeb** [comment]
@tdipisa 
We need to solve this before we go into production - so I kindly ask you to let me know if this can be fixed in code or if the freemarker template are to be changed (and what has to be changed)

**Comment by dsuren1** [comment]
@kgaeb 
Actually, it looks like during deployment on your end, as certain style classes are added that supports styling of GFI data render in html viewer. 

Here I checked in that DEV env (if i'm not wrong), https://sdimd-free.austrocontrol.at/mapstore/#/
The following style classes (`.equal`, `.my-auto`) are added as part of configuration at your end during deployment.
![image](https://user-images.githubusercontent.com/26929983/193555208-c92fdbba-663f-4478-8bbe-6a64cb1259ac.png)
These classes are required for proper render of the GFI content in html viewer. 
**Note:** This will **not be present** in the test env https://austrocontrol.geo-solutions.it/austrocontrol-ms2/#/

When I added these to my local, I was able to get the same result as DEV & PROD.

_Kindly check if these classes are added at your end during deployment to respective environments_
